assets/base.css

```diff
@@ -1449,7 +1449,15 @@ details[open] > .share-button__fallback {
   z-index: 3;
   width: 100%;
   min-width: max-content;
-  box-shadow: 0 0 0 0.1rem rgba(var(--color-foreground), 0.55);
+  border: 0.1rem solid transparent;
+  border-radius: var(--inputs-radius);
+  box-shadow: 0 0 0 var(--inputs-border-width) rgba(var(--color-foreground), var(--inputs-border-opacity));
+  filter: drop-shadow(
```

Comment:
Was there a reason I missed that we didn't update this shadow? If we add it as box-shadow instead of the filter we can probably avoid the weird effects. Though after making that change we may have to move this to a pseudo element so we can properly scale the shadow dimensions to account for thicker borders.
<img width="363" alt="Screen Shot 2022-01-14 at 10 03 03 AM" src="https://user-images.githubusercontent.com/30790058/149555390-770ad1f7-4946-4507-9743-40bd25e74d9a.png">

Reply:
Good catch! Refactored to use the `after` and `before` as well: https://github.com/Shopify/dawn/pull/1097/commits/6a223875d14de4daaf4c978f9fa1a67240b0539f